### DISCUSSION TITLE
Windhawk make Virtual Box crashes "Error in supR3HardNtChildPurify"

### DISCUSSION THREAD

**Discussion by Ecolohy** [ramensoftware/windhawk#336] [General]
Hello together
I installaed on my new PC with win11 24h2 Windhawk and everything was fine till I like to test something in the Virtual-Machine (Oracle Virtual Box ver. 7.1.4).
The VB wont start anymore and gave me an Error in supR3HardNtChildPurify. -> See logfile...
I do had to terminate Windhawk to start the VB again.
Why is this happen and is there a way to have both running?
Thanks!

![Screenshot 2024-12-02 091543](https://github.com/user-attachments/assets/85346365-9053-47a8-b363-2db125da254c)
[VBoxHardening.log](https://github.com/user-attachments/files/17974805/VBoxHardening.log)

**Comment by m417z** [comment]
Hi, you can exclude VirtualBox, see my comment here: https://github.com/ramensoftware/windhawk/issues/25#issuecomment-1338915056.

**Reply by Ecolohy** [reply]
It works! Whohooo! Happy to here that! Many thanks. I was searching for a while to find out, that Windhawk was crashing VB....!
